ci_framework/roles/build_containers/tasks/install.yml

```diff
@@ -20,8 +20,9 @@
   # FIXME(Chandan/arxcruz): Skip usage of sudo installation
 - name: Install tcib package
   become: true
-  ansible.builtin.shell: |
-    pip install -r requirements.txt
-    python setup.py install
-  args:
+  ci_script:
```

Comment:
@IgnacioPro We should drop ci_script from build_containers role, It is breaking currently. Based on my testproject https://review.rdoproject.org/r/c/testproject/+/49693 and log link https://logserver.rdoproject.org/93/49693/3/check/periodic-container-tcib-build-push-centos-9-antelope/1c41dc0/job-output.txt
```
ASK [Build Containers name=build_containers] **********************************
2023-08-28 09:13:18.113494 | ERROR! couldn't resolve module/action 'ci_script'. This often indicates a misspelling, missing collection, or incorrect module path.
2023-08-28 09:13:18.113499 | 
2023-08-28 09:13:18.113504 | The error appears to be in '/var/lib/zuul/builds/1c41dc0de4e0457d9cf5f973cbb9f039/untrusted/project_1/github.com/openstack-k8s-operators/ci-framework/ci_framework/roles/build_containers/tasks/install.yml': line 21, column 3, but may
2023-08-28 09:13:18.113508 | be elsewhere in the file depending on the exact syntax problem.
2023-08-28 09:13:18.113513 | 
2023-08-28 09:13:18.113518 | The offending line appears to be:
2023-08-28 09:13:18.113522 | 
2023-08-28 09:13:18.113527 |   # FIXME(Chandan/arxcruz): Skip usage of sudo installation
2023-08-28 09:13:18.113532 | - name: Install tcib package
2023-08-28 09:13:18.113536 |   ^ here
```

Reply:
hmmm didn't we solve the same kind of issue for the `ci_make` ?!

Reply:
I am fine with both approaches.

Reply:
@IgnacioPro Can we drop ci_script from build_containers role and get this pr merged?

Reply:
@cjeanner Nope, we were not able to fix `ci_make` issue, Zuul is not able to discover the issue.

Reply:
'k. so yeah, @IgnacioPro please discard the changes in build_containers role (for now).

Reply:
@cjeanner @raukadah on it!